examples/local/README.md

```diff
@@ -0,0 +1,204 @@
+# Running aws-operator Locally
+
+**Note:** This should only be used for testing and development. See the
+[/kubernetes/][kubernetes-dir] directory and [Secrets][secrets-doc] for
+a production ready configuration.
+
+[kubernetes-dir]: https://github.com/giantswarm/aws-operator/tree/master/kubernetes
+[secrets-doc]: https://github.com/giantswarm/aws-operator#secret
+
+This guide explains how to get running aws-operator locally. For example on
+minikube. Also how to create a managed Kubernetes cluster with single master
+and single worker using the operator.
+
+All commands are assumed to be run from `examples/local` directory.
+
+
+## Preparing Templates
+
+All yaml files in this directory are templates. Before proceeding this guide
+all placeholders must be replaced with sensible values.
+
+- *CLUSTER_NAME* - Cluster's name.
+- *COMMON_DOMAIN* - Cluster's etcd and API common domain.
+- *COMMON_DOMAIN_INGRESS* - Ingress common domain.
+- *ID_RSA_PUB* - SSH public key to be installed on nodes.
+- *AWS_ACCESS_KEY_ID* - AWS access key.
+- *AWS_SECRET_ACCESS_KEY* - AWS secret.
+- *AWS_REGION* - AWS region.
+- *AWS_AZ* - AWS availability zone.
+- *AWS_AMI* - AWS image to be used on both master and worker machines.
+- *AWS_INSTANCE_TYPE_MASTER* - Master machines instance type.
+- *AWS_INSTANCE_TYPE_WORKER* - Worker machines instance type.
+
+Below is handy snippet than can be used to make that painless. It works in bash and zsh.
+
+```bash
+export CLUSTER_NAME="example-cluster"
+export COMMON_DOMAIN="internal.company.com"
+export COMMON_DOMAIN_INGRESS="company.com"
+export ID_RSA_PUB="$(cat ~/.ssh/id_rsa.pub)"
+export AWS_ACCESS_KEY_ID="AKIAIXXXXXXXXXXXXXXX"
+export AWS_SECRET_ACCESS_KEY="XXXXXXXXXXXXXXXXX/XXXXXXXXXXXXXXXXXXXXXX"
+export AWS_REGION="eu-central-1"
+export AWS_AZ="eu-central-1a"
+export AWS_AMI="ami-d60ad6b9"
+export AWS_INSTANCE_TYPE_MASTER="t2.medium"
+export AWS_INSTANCE_TYPE_WORKER="t2.medium"
+
+for f in *.tmpl.yaml; do
+    sed \
+        -e 's\${CLUSTER_NAME}\'"${CLUSTER_NAME}"'\g' \
+        -e 's\${COMMON_DOMAIN}\'"${COMMON_DOMAIN}"'\g' \
+        -e 's\${COMMON_DOMAIN_INGRESS}\'"${COMMON_DOMAIN_INGRESS}"'\g' \
+        -e 's\${ID_RSA_PUB}\'"${ID_RSA_PUB}"'\g' \
+        -e 's\${AWS_ACCESS_KEY_ID}\'"${AWS_ACCESS_KEY_ID}"'\g' \
+        -e 's\${AWS_SECRET_ACCESS_KEY}\'"${AWS_SECRET_ACCESS_KEY}"'\g' \
+        -e 's\${AWS_REGION}\'"${AWS_REGION}"'\g' \
+        -e 's\${AWS_AZ}\'"${AWS_AZ}"'\g' \
+        -e 's\${AWS_AMI}\'"${AWS_AMI}"'\g' \
+        -e 's\${AWS_INSTANCE_TYPE_MASTER}\'"${AWS_INSTANCE_TYPE_MASTER}"'\g' \
+        -e 's\${AWS_INSTANCE_TYPE_WORKER}\'"${AWS_INSTANCE_TYPE_WORKER}"'\g' \
+        ./$f > ./${f%.tmpl.yaml}.yaml
+done
+```
```

Comment:
I'm getting errors running this. This is on Mac, I use zsh but I tried bash too.

```
sed: 1: "s\${CLUSTER_NAME}\ross\g
": substitute pattern can not be delimited by newline or backslash
```